src/content/app/genome-browser/components/track-panel/trackPanelConfig.ts

```diff
@@ -54,7 +54,12 @@ export type TrackStates = {
 
 export type GenomeTrackStates = {
   commonTracks?: TrackStates;
-  objectTracks?: { [objctId: string]: TrackStates };
+  objectTracks?: { [objectId: string]: TrackStates };
+  transcriptTracks?: {
+    [objectId: string]: {
+      [transcriptId: string]: TrackActivityStatus;
+    };
+  };
```

Comment:
This is something to think about — whether it's the appropriate way of representing the states of the transcripts of a focus gene. I wonder what alternative ways of representing them we can think of.